staff/xavi-gonzalez/git/README.md

```diff
@@ -0,0 +1,146 @@
+# Git Commands
+
+Command-line interface commands typical in GIT.
+
+## git init 
+creates a new local repository in the directory you are in
+```sh
+$ git init
+```
+
+## git status
+shows the status of your files in the directory you are working in
+
+## git clone
+makes a copy of an existing remote repository
+```sh
+$ git clone "repository name"
+```
+
+## git pull
+downloads the latest commits from the remote repository.
+```sh
+$ git pull "repository name"
+```
+
+## git merge
+allows merging changes from one branch into another branch, creating a new commit if no problem is found
+```sh
+$ git merge
+```
+
+## git diff
+shows the differences between 2 commits, branches, files, etc
+```sh
+$ git diff
+diff --git a/staff/xavi-gonzalez/git/README.md b/staff/xavi-gonzalez/git/README.md
+index e69de29..3829526 100644
+--- a/staff/xavi-gonzalez/git/README.md
++++ b/staff/xavi-gonzalez/git/README.md
+```
+
+## git log 
+check commit status
+```sh
+$ git log
+
+commit 3d5f56bfc51f2e5c677bc6437e5d8ddea48451ef (HEAD -> feature/git, origin/feature/git)
+```
+
+## git branch
+check current branch
+```sh
+$ git branch
+
+  develop
+  feature/arrays
+* feature/bash
+  feature/complex-structure
+  feature/playground
+  feature/space-invaders
+  feature/strings
+  main
+```
+
+## git branch -D
+delete a branch
+```sh
+$ git branch feature/branch name to remove
+```
+
+
+## git branch "new branch name"
+creates a new branch
+```sh
+$ git branch feature/branch name to create
+```
+
+## git checkout 
+access a different branch
+```sh
+$ git checkout feature/playground
+```
+
+## git add
+add new file/folder ready to commit & push
+```sh
+$ git add staff/xavi-gonzalez/git   
+```
+
+## git commit -m
+prepares file to be uploaded with an specific written command
+```sh
+$ git commit -m 'add git #162'
+```
+
+## git push
+upload new commit
```

Comment:
The command implementation should appear inside a back ticks section, just as the other command examples 🙀

Reply:
ok done. Again, you are the best teacher